yt_dlp/postprocessor/embedthumbnail.py

```diff
@@ -88,14 +88,39 @@ def run(self, info):
 
         mtime = os.stat(encodeFilename(filename)).st_mtime
 
+        avoid_mutagen = any(
+            opt in self.get_param('compat_opts', [])
+            for opt in ('avoid-mutagen', 'embed-thumbnail-atomicparsley'))
```

Comment:
if `embed-thumbnail-atomicparsley` is just an alias for `avoid-mutagen`, is all of this here necessary?

or could we do this?
```suggestion
        avoid_mutagen = 'avoid-mutagen' in self.get_param('compat_opts', [])
```

Reply:
It's needed for API compat